src/utils/utils.js

```diff
@@ -0,0 +1,7 @@
+import axios from "axios";
+import {apiAddress} from './apiAddress.json';
+
+export const getFromApi = (endpoint) => {
+    return axios.get(`${apiAddress}${endpoint}`)
+}
```

Comment:
interesting approach.  I would prefer to have this utils file have different calls to the API for each endpoint.  But this still looks okay to my eyes.   You will need a different function for the other CRUD operations though.  It will be interesting to see if this approach will be able to handle queries later as well!

Reply:
Oh true, yes I didn't think about bundling them into one. thats a better idea